support-frontend/app/services/stepfunctions/SupportWorkersClient.scala

```diff
@@ -33,6 +33,7 @@ object CreateSupportWorkersRequest {
   implicit val giftRecipientCodec: Codec[GiftRecipientRequest] = deriveCodec
 
   implicit val decoder: Decoder[CreateSupportWorkersRequest] = deriveDecoder[CreateSupportWorkersRequest]
+  implicit val encoder: Encoder[CreateSupportWorkersRequest] = deriveEncoder[CreateSupportWorkersRequest]
```

Comment:
you should be able to use Codec if you need both a Decoder and Encoder

Reply:
Good shout